charts/librespeed/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+appVersion: 1.0.6-12
+description: Librespeed is a HTML5 webpage to test upload and download speeds
+name: librespeed
+version: 1.0.0
+keywords:
+  - Librespeed
+home: https://github.com/billimek/billimek-charts/tree/master/charts/librespeed
+icon: https://github.com/librespeed/speedtest/blob/master/.logo/logo3.png?raw=true
+sources:
+  - https://github.com/librespeed/speedtest
+  - https://hub.docker.com/r/linuxserver/librespeed
+  - https://github.com/billimek/billimek-charts/tree/master/charts/Librespeed
```

Comment:
This is really minor, but can we make this instance lowercase (e.g. `librespeed`)